net.zgcoder.skycheckers.json

```diff
@@ -0,0 +1,136 @@
+{
+	"app-id": "net.zgcoder.skycheckers",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "19.08",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "skycheckers",
+	"finish-args": [
+		"--socket=x11", "--share=ipc",
+		"--device=dri", "--socket=pulseaudio", "--share=network", "--device=all"
+	],
+	"modules": [
+		{
+			"name": "SDL2",
```

Comment:
Same as glew

Reply:
It's not obvious where SDL2, mixer, ttf shared modules are. (I need/want 2.0.12 for SDL2 btw).

Reply:
Ok I presume they're built in to the runtime (not obvious :p). I removed SDL2_mixer and SDL2_ttf then. Still keeping SDL until I know for sure I can have >= 2.0.12.